editor/src/components/canvas/canvas-strategies/strategies/flow-reorder-helpers.ts

```diff
@@ -75,8 +75,10 @@ function areNonWrappingSiblings(
   })
 }
 
-function getElementDirection(element: ElementInstanceMetadata): 'vertical' | 'horizontal' {
-  const displayValue = element.specialSizeMeasurements.display
+export function getElementDirection(
+  element: ElementInstanceMetadata | null,
+): 'vertical' | 'horizontal' {
+  const displayValue = element?.specialSizeMeasurements.display
   return displayValue === 'inline' || displayValue === 'inline-block' ? 'horizontal' : 'vertical'
```

Comment:
not part of this PR, but we should actually use `displayValue.includes('inline')` to also include inline-flex and inline-grid and whatever else
```suggestion
  return displayValue.includes('inline') ? 'horizontal' : 'vertical'
```

Reply:
👍